rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+# Lightweight RFC Process
+
+**Geode-RFC-000**
```

Comment:
Can we not zero pad? I feel like that zero padding makes it less readable. IETF RFCs don't zero pad.

Reply:
Sounds good 👍🏻. Thoughts on keeping the file zero padded for sorting purposes?

Reply:
I would rather the file name match the title, no padding. Sorting by numerical order doesn't feel like it helps in finding what you want. The numbers are rather arbitrary to me.

Reply:
I think the value of sorting by number might come once we are a few years in and have a fair number of superseded proposals.